samples/audio/samples.html

```diff
@@ -503,6 +503,20 @@
 </tr> 
 
 
+<tr>
+<td style="width: 60px;">
+<div style="display: block; text-align: left;"><a href="http://acompas.audio" imageanchor="1"><img src="images/acompas.png" border="1"></a></div>
+</td>
+<td style="width: 3000px;"><a href="http://acompas.audio">
```

Comment:
Can you put `target="_blank"` in the `<a>` element?

Sorry for asking this, but eventually this whole page will be cleaned up at some point.